ext/opcache/Optimizer/sccp.c

```diff
@@ -1035,8 +1035,8 @@ static void sccp_visit_instr(scdf_ctx *scdf, zend_op *opline, zend_ssa_op *ssa_o
 				break;
 			}
 
-			/* We're only interested in functions with one or two arguments right now */
-			if (call->num_args == 0 || call->num_args > 2) {
+			/* We're only interested in functions with one, two or three arguments right now */
+			if (call->num_args == 0 || call->num_args > 3) {
```

Comment:
There is a `zval *args[2]` declaration higher up in the code that needs to be synchronized with this change (or just switched to alloca instead).